snippets/rosa-classic-support.adoc

```diff
@@ -0,0 +1,10 @@
+// When including this file, ensure that {FeatureName} is set immediately before
+// the include. Otherwise it will result in an incorrect replacement.
+
+[IMPORTANT]
+====
+[subs="attributes+"]
+{FeatureName} is not supported on {product-title} (ROSA) with hosted control planes (HCP).
```

Comment:
Same question re: potential HCP attribute usag, altho I suppose it's a little trickier here, ensuring the correct attributes file is included in whichever module/assembly the snippet might appear. Presumably it would be for ROSA content, tho... 🤷